demosplan/DemosPlanCoreBundle/Repository/SupportContactRepository.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of the package demosplan.
+ *
+ * (c) 2010-present DEMOS plan GmbH, for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+namespace demosplan\DemosPlanCoreBundle\Repository;
+
+class SupportContactRepository extends FluentRepository
```

Comment:
Is the whole class needed if all actions are done via the resource type? :thinking: 
Im note sure at all.

Reply:
The ResourceType needs the repository extending the FluentRepository